manager/state/raft/raft.go

```diff
@@ -1647,7 +1652,7 @@ func getIDs(snap *raftpb.Snapshot, ents []raftpb.Entry) []uint64 {
 		case raftpb.ConfChangeUpdateNode:
 			// do nothing
 		default:
-			log.G(context.Background()).Panic("ConfChange Type should be either ConfChangeAddNode or ConfChangeRemoveNode!")
+			log.G(ctx).Panic("ConfChange Type should be either ConfChangeAddNode or ConfChangeRemoveNode!")
```

Comment:
I wonder if we should just make these normal `panic`s. Having to pass a context to `getIDs` is kind of annoying.

Or we could use `log.L` instead of `log.G(ctx)`. I don't think using the logger from the context adds any value in these situations.